_jobs/software-engineer.md

```diff
@@ -10,52 +10,61 @@ Compiler is a small, woman-owned software consultancy. Compiler helps service-or
+Our clients have included: City of Los Angeles, City of West Hollywood, CicLAvia, LA2050, Caltrans, County of Los Angeles, Southern California Association of Governments, and the California Community Foundation. Our recent work includes building and integrating payment and digital services technologies for public transportation agencies using Docker, Python/Django/Flask, and Azure.
 
-We are looking for a Software Engineer to support a range of new and ongoing client projects. You’ll report directly to Kegan Maher and collaborate with a small team of designers and engineers. You’ll have access to additional external support for data privacy, data security, accessibility, and internationalization as well as modern tools to help you get your work done in a remote environment.
+We are looking for a Software Engineer to support a range of new and ongoing client projects. You’ll report directly to Kegan Maher and collaborate closely with a small team of product managers, designers, and engineers. You’ll have access to additional external support for data privacy, data security, accessibility, and internationalization as well as modern tools to help you get your work done in a remote environment.
+
+This is a full-time, salaried position (W-2) with benefits. Full-time employment at Compiler is thirty (30) hours per week.
 
 ## Responsibilities
 
 - Design and build open source software that streamlines the delivery of public services with a focus on equity and active harm reduction
 - Interact with clients to iteratively gather requirements, refine scope, and demo progress
 - Document your work, including asynchronous communications, to inform internal and client teams
-- Coordinate with product, design, and engineering to deliver delightful products and services
+- Coordinate with product, design, and engineering to deliver delightful products and services that meet current Web Content Accessibility Guidelines (WCAG)
 - Pair program, code review, contribute to architecture decisions, help create standard patterns and practices for development and DevOps
-- Self-direct and time-manage across multiple client projects with varying levels of complexity, demand, and constraint
+- Self-direct and time-manage across multiple client and internal projects with varying levels of complexity, demand, and constraint
 
-## Preferred qualifications
+## Preferred Qualifications
 
 - Experience working with or in proximity to government
 - Experience using modern, high-level programming languages such as Python and JavaScript to build web applications, API services, or similar
+- Experience developing performant and accessible front-end components that meet WCAG standards, using advanced HTML and CSS
```

Comment:
```suggestion
- Experience developing performant and accessible front-end components that meet WCAG standards, using advanced HTML and CSS
- Experience maintaining databases
- Experience working with external APIs
```

Reply:
These are both great suggestions but I feel like as worded, they aren't saying enough. Can you please add a bit more to each of these new bullets? 

E.g. `Experience integrating production applications with external/3rd party APIs to do X (or for reason Y)`

Reply:
Updated my suggestion